docs/management/connectors/action-types/index.asciidoc

```diff
@@ -26,8 +26,7 @@ image::management/connectors/images/index-connector.png[Index connector]
 [[index-connector-configuration]]
 ==== Connector configuration
 
-Index connectors must have a name and an {es} index. You can optionally set the
```

Comment:
@lcawl I removed references to `refresh` in the docs but did not touch the API as we are not changing the API (no breaking changes), just removing the ability to set the value in the UI.

Reply:
I also tried to run the screenshot generation tests to regenerate a screenshot but got an error